### DISCUSSION TITLE
How to add sample level weight for the loss function?

### DISCUSSION THREAD

**Discussion by ajing** [Lightning-AI/pytorch-lightning#13270] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Is there any easy way to do it in pytorch lightning?
